app/src/main/java/com/mailerdaemon/app/PlacementFragment.kt

```diff
@@ -0,0 +1,34 @@
+package com.mailerdaemon.app
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.mailerdaemon.app.placement.PlacementActivity
+import com.mailerdaemon.app.placement.PlacementAdapter
+import com.mailerdaemon.app.placement.PlacementModel
+import kotlinx.android.synthetic.main.activity_placement.*
+import kotlinx.android.synthetic.main.fragment_placement.*
+import kotlinx.android.synthetic.main.fragment_placement.view.*
+
+class PlacementFragment : Fragment() {
+
+    var data: List<PlacementModel>? = null
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        val view = inflater.inflate(R.layout.fragment_placement, container, false)
+
+        data = arguments?.getParcelableArrayList("data")
```

Comment:
I realized getting data from argument is not right. The data is passed once and to update it we need to push fragment again. Instead Make a new Interface `ShowData` with method `showData()` and implement it in Fragment. Also make set an argument for getDatabase() in activity ( change `getDatabase()` to `getDatabase(showData: ShowData?)`) . Call `getDatabse()' method in fragment when you need to show data. The interface will serve as a callback. So call `showDatabase()` when you recieve data in `getDatabase()`.